src/models/index.ts

```diff
@@ -15,7 +15,7 @@ const dbURLS: DBInterface = {
   development: DATABASE_URL,
 };
 
-const URL: string = dbURLS[NODE_ENV] as string;
+const URL: string = dbURLS['development'] as string;
```

Comment:
Why are you changing this @alainburindi ?
Read the app environment form the command line!
This is what is making your tests failing when running on CI!
On ci, we are in `test` environment